charts/tekton-pipelines/values.yaml

```diff
@@ -20,13 +20,7 @@ controller:
                   - windows
   tolerations: []
   nodeSelector: {}
-  resources:
```

Comment:
We trying to avoid changes in 3party chats as much as possible. Is there any reason of removing the defualt values?

Reply:
Yes. They can stay in there